TheMET/TheMET/FeaturedArtWorkLoadingView/LoadingPlaceholderView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LoadingPlaceholderView.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 18/04/2023.
+//
+
+import Foundation
+import UIKit
+
+class LoadingPlaceholderView: UIView {
+    
+    static let xibFileName = "LoadingPlaceholderView"
+    
+    @IBOutlet var textLabel: UILabel!
+    @IBOutlet var imageView: UIImageView!
+    @IBOutlet var button: UIButton!
+    
+    static func constructView() -> LoadingPlaceholderView {
+        let nib = UINib(nibName: LoadingPlaceholderView.xibFileName, bundle: nil)
+        return nib.instantiate(withOwner: nil).first as! LoadingPlaceholderView
+        }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    func setup(configuration: LoadingPlaceholderConfiguration) {
```

Comment:
I suggest to make this method `private`, add an argument `configuration: LoadingPlaceholderConfiguration` to `constructView()` method and execute `setup` there on the constructed object

Reply:
fixed